lib/thinking_sphinx/real_time/populator.rb

```diff
@@ -12,9 +12,9 @@ def populate(&block)
 
     remove_files
 
-    scope.find_each do |instance|
-      transcriber.copy instance
-      instrument 'populated', :instance => instance
+    scope.find_in_batches do |instances|
+      transcriber.copy instances
+      instrument 'populated', :instance => instances
```

Comment:
I forgot to mention that I wasn't sure what the instance option was for and if anything was using it so I just dumped the instances to it (I didn't see any references, but didn't want to rename it if something was logging it)